### PR TITLE
Feat(Task 12): adds task 12, its solution and some QOL

### PR DESCRIPTION
**Pull request by SSmale**
redo of #24

### CODE REVIEW

File: src/workspace/js/tasks.json [hunk 1]
```diff
@@ -1,56 +1,73 @@
...
+        "op": "/solutions/task11.js"
+    },
+
+    {
+        // Task 12
+        "task": "Keep up the good work! The next topic we will cover is functions. Functions are sections of code that exicute other code and make it reuseable. \nIn ES6 there are 2 main ways to declare a function, the function keyword and arrow functions.\n Syntax: function name(params) {\n   code\n} \nTask 12: Declare an function using the keyword that when given 2 numbers will add them together, then print out the result. The number pairs you need to test are 1+2, 10+15, 100+400",
```
Comment (jamesgeorge007) [inline src/workspace/js/tasks.json]:
```suggestion
        "task": "Keep up the good work! The next topic we will cover is functions. Functions are sections of code that execute other code and make it reusable. \nIn ES6 there are 2 main ways to declare a function, the function keyword and arrow functions.\n Syntax: function name(params) {\n   code\n} \nTask 12: Declare a function using the respective keyword that when given 2 numbers will add them together, then print out the result. The number pairs you need to test are 1+2, 10+15, 100+400",
```

Comment (SSmale) [reply src/workspace/js/tasks.json]: Is there another way to track which task is which, it makes it quite difficult to follow without any reference points.

Comment (jamesgeorge007) [reply src/workspace/js/tasks.json]: We're having the `solution` key which serves the purpose.

Comment (jamesgeorge007) [reply src/workspace/js/tasks.json]: You still haven't fixed those typos. Just apply the commit suggestion :point_up:

**Pull request merged**